src/main/java/org/mdbenefits/app/data/enums/HelpNeededType.java

```diff
@@ -3,24 +3,45 @@
 import static org.mdbenefits.app.data.enums.ProgramType.SNAP;
 import static org.mdbenefits.app.data.enums.ProgramType.TCA;
 import static org.mdbenefits.app.data.enums.ProgramType.TDAP;
-import static org.mdbenefits.app.data.enums.ProgramType.OTHER;
+import static org.mdbenefits.app.data.enums.ProgramType.RCA;
+
+import java.util.HashMap;
+import java.util.List;
+
+import java.util.ArrayList;
+import java.util.Map;
 
 import lombok.Getter;
 
 @Getter
 public enum HelpNeededType {
 
-  FOOD( "help-needed.food", SNAP),
-  CHILDREN("help-needed.children", TCA),
-  //UTILITIES("help-needed.utilities", OHEP),
-  DISABILITY("help-needed.disability", TDAP),
-  NOT_SURE("help-needed.not-sure", OTHER);
+    FOOD("help-needed.food", SNAP, new ArrayList<>(List.of(SNAP, TCA))),
+    CHILDREN("help-needed.children", TCA, new ArrayList<>(List.of(TCA, SNAP))),
+    //UTILITIES("help-needed.utilities", OHEP,),
+    DISABILITY("help-needed.disability", TDAP, new ArrayList<>(List.of(TDAP))),
+    REFUGEE("help-needed.refugees", RCA, new ArrayList<>(List.of(RCA)));
+
+    private final String labelSrc;
+    private final ProgramType programType;
+    private final ArrayList<ProgramType> relevantPrograms;
+
+    static private final Map<String, ArrayList<ProgramType>> MAP_BY_PROGRAM_TYPE = new HashMap<>();
+
+    static {
+        for (HelpNeededType type : HelpNeededType.values()) {
+            MAP_BY_PROGRAM_TYPE.put(type.programType.toString(), type.relevantPrograms);
+        }
+    }
 
-  private final String labelSrc;
-  private final ProgramType programType;
+    HelpNeededType(String labelSrc, ProgramType programType, ArrayList<ProgramType> relevantPrograms) {
+        this.labelSrc = labelSrc;
+        this.programType = programType;
+        this.relevantPrograms = new ArrayList<>(relevantPrograms);
+    }
 
-  HelpNeededType(String labelSrc, ProgramType programType) {
-    this.labelSrc = labelSrc;
-    this.programType = programType;
-  }
+    public static ArrayList getRelevantProgramsFromValue(String programType) {
+        ArrayList<ProgramType> relevantPrograms = MAP_BY_PROGRAM_TYPE.get(programType);
+        return relevantPrograms;
+    }
```

Comment:
We could have something that you can provide the name and it returns the programValues

Map By ENUM Name